client/src/main/java/io/pravega/client/admin/impl/StreamManagerImpl.java

```diff
@@ -104,6 +113,14 @@ public boolean deleteScope(String scopeName) {
                 RuntimeException::new);
     }
 
+    @Override
+    @SuppressWarnings("deprecation")
```

Comment:
I know you are keeping the code in the `batch` package to avoid the duplication, but shouldn't we do the other way around, bring the code under `admin` because:

1- We avoid such deprecation warnings altogether
2- This is where the code is supposed to live going forward (until the next refactoring)

what do you think?

Reply:
The code to fetch the `StreamCut`s for the HEAD and TAIL of the stream are reused by the batchclient api  `io.pravega.client.batch.impl.BatchClientImpl#getSegments`. Hence moving the code to StreamManagerImpl will not remove code duplication.

Reply:
Right, that's why @tkaitchuck suggested to create some helper class to enable the common code to be shared. It is not a big deal, but it might simplify our work when we need to remove the deprecated call.

Reply:
yes, makes sense. Refactored the code to address your comment.